environment.yml

```diff
@@ -26,6 +26,7 @@ dependencies:
   - click
   - typing-extensions
   - lomap2>=2.3.0
+  - openmm>=8.0.0
```

Comment:
These solvation settings need OpenMM 8.0. Overall supporting multiple major versions of OpenMM is probably unwise as we'll need to test them all so I would suggest bumping up the minimum version?

Otherwise we can add a version check to the validator.